uber/models.py

```diff
@@ -1038,7 +1038,7 @@ def _misc_adjustments(self):
         if c.AT_THE_CON and self.badge_num and self.is_new:
             self.checked_in = datetime.now(UTC)
 
```

Comment:
I hadn't realized it, but since we already set a default value of ``c.AGE_UNKNOWN`` this was never getting invoked.  So even though we've been tracking birthdates, the age group hasn't been getting properly displayed in some places.